commands/service_dev.go

```diff
@@ -48,21 +50,19 @@ func (c *serviceDevCmd) preRunE(cmd *cobra.Command, args []string) error {
 }
 
 func (c *serviceDevCmd) runE(cmd *cobra.Command, args []string) error {
-	var (
-		id    string
-		valid bool
-		err   error
-	)
-	pretty.Progress("Deploying the service...", func() {
-		id, valid, err = c.e.ServiceDeploy(c.path)
-	})
+	statuses := make(chan provider.DeployStatus, 0)
```

Comment:
Small change, but if you're putting `0`, just skip it and write `make(chan provider.DeplyStatus)`

```warning: should use make(chan provider.DeployStatus) instead (S1019) (megacheck)```